interfaces/cython/cantera/yaml2ck.py

```diff
@@ -0,0 +1,714 @@
+# This file is part of Frhodo. Copyright © 2020, UChicago Argonne, LLC
+# and licensed under BSD-3-Clause. See License.txt in the top-level 
+# directory for license and copyright information.
+
+# Adapted from Kyle Niemeyer's pyMARS Jul 24, 2019
+
+# Writes a solution object to a chemkin inp file
+# currently only works for Elementary, Falloff and ThreeBody Reactions
+
+# KE Niemeyer, CJ Sung, and MP Raju. Skeletal mechanism generation for surrogate fuels using directed relation graph with error propagation and sensitivity analysis. Combust. Flame, 157(9):1760--1770, 2010. doi:10.1016/j.combustflflame.2009.12.022
+# KE Niemeyer and CJ Sung. On the importance of graph search algorithms for DRGEP-based mechanism reduction methods. Combust. Flame, 158(8):1439--1443, 2011. doi:10.1016/j.combustflflame.2010.12.010.
+# KE Niemeyer and CJ Sung. Mechanism reduction for multicomponent surrogates: A case study using toluene reference fuels. Combust. Flame, in press, 2014. doi:10.1016/j.combustflame.2014.05.001
+# TF Lu and CK Law. Combustion and Flame, 154:153--163, 2008. doi:10.1016/j.combustflame.2007.11.013
+
+from mailbox import NotEmptyError
+import os, pathlib, sys
+from textwrap import fill
+from collections import Counter
+import getopt
+
+import cantera as ct
+
+# number of calories in 1000 Joules
+CALORIES_CONSTANT = 4184.0
+
+# Conversion from 1 debye to coulomb-meters
+DEBEYE_CONVERSION = 3.33564e-30
```

Comment:
not a big deal, but it's "Debye", not "Debeye"

Reply:
Corrected